### PR TITLE
lrzuntar: add page

### PR DESCRIPTION
**Pull request by russsaidwords**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/linux/lrzuntar.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# lrzuntar
+
+> A wrapper for `lrunzip` to simplify decompression of directories.
+> See also: `lrztar`, `lrzip`.
+
+- Decompress from file to a directory:
```
Comment (sbrl) [inline pages/linux/lrzuntar.md]:
```suggestion
- Decompress from a file to a directory:
```

Perhaps this reads better?

File: pages/linux/lrzuntar.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`lrzuntar {{path/to/directory.tar.lrz}}`
+
+- Decompress from file to a directory using a specific number of processor threads:
```
Comment (sbrl) [inline pages/linux/lrzuntar.md]: Same here.

File: pages/linux/lrzuntar.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`lrzuntar -p {{8}} {{path/to/directory.tar.lrz}}`
+
+- Decompress from file to a directory and silently overwrite items that already exist:
```
Comment (sbrl) [inline pages/linux/lrzuntar.md]: Same here.

File: pages/linux/lrzuntar.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`lrzuntar -f {{directory.tar.lrz}}`
+
+- Specify output path:
```
Comment (sbrl) [inline pages/linux/lrzuntar.md]:
```suggestion
- Specify the output path:
```

Also, `directory.tar.lrz` -> `path/to/output_archive.tar.lrz`?

File: pages/linux/lrzuntar.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+`lrzuntar -O {{path/to/directory}} {{directory.tar.lrz}}`
+
+- Delete compressed file after decompressing to directory:
```
Comment (sbrl) [inline pages/linux/lrzuntar.md]:
```suggestion
- Delete the compressed file after decompression:
```

Also, perhaps `path/to/directory.tar.lrz` -> `path/to/archive.tar.lrz`?

File: pages/linux/lrzuntar.md (hunk 4 above)
Comment (russsaidwords) [reply pages/linux/lrzuntar.md]: The output path in this instance is the directory that the decompressed directory will appear within.

Comment (sbrl) [reply pages/linux/lrzuntar.md]: Ah, ok.

**Comment by sbrl** [comment]
Thanks again, @russsaidwords :smiley_cat:

**Pull request merged**